src/DeviceInterface/InterfacePrototype.js

```diff
@@ -130,29 +133,74 @@ class InterfacePrototype {
         }
     }
 
-    attachTooltip (form, input) {
+    selectedDetail (data, type) {
+        this.activeFormSelectedDetail(data, type)
+    }
+
+    activeFormSelectedDetail (data, type) {
+        const form = this.getActiveForm()
```

Comment:
This allows us to have reuse in the #49 patch so the apple interface can call this.